salt/hdp/templates/cfg_pico.py.tpl

```diff
@@ -179,56 +179,57 @@ BLUEPRINT = r'''{
+                    "yarn.scheduler.capacity.root.applications.acl_submit_applications" : " ",
+                    "yarn.scheduler.capacity.root.applications.capacity" : "1",
+                    "yarn.scheduler.capacity.root.applications.dev.acl_administer_queue" : " dev,prod",
+                    "yarn.scheduler.capacity.root.applications.dev.acl_submit_applications" : " dev,prod",
+                    "yarn.scheduler.capacity.root.applications.dev.capacity" : "1",
+                    "yarn.scheduler.capacity.root.applications.dev.maximum-applications" : "100000000",
+                    "yarn.scheduler.capacity.root.applications.dev.maximum-capacity" : "100",
+                    "yarn.scheduler.capacity.root.applications.dev.minimum-user-limit-percent" : "100",
+                    "yarn.scheduler.capacity.root.applications.dev.ordering-policy" : "fair",
+                    "yarn.scheduler.capacity.root.applications.dev.ordering-policy.fair.enable-size-based-weight" : "false",
+                    "yarn.scheduler.capacity.root.applications.dev.priority" : "0",
+                    "yarn.scheduler.capacity.root.applications.dev.state" : "RUNNING",
+                    "yarn.scheduler.capacity.root.applications.dev.user-limit-factor" : "10000",
+                    "yarn.scheduler.capacity.root.applications.maximum-capacity" : "100",
+                    "yarn.scheduler.capacity.root.applications.minimum-user-limit-percent" : "100",
+                    "yarn.scheduler.capacity.root.applications.priority" : "0",
+                    "yarn.scheduler.capacity.root.applications.prod.acl_administer_queue" : " prod",
+                    "yarn.scheduler.capacity.root.applications.prod.acl_submit_applications" : " prod",
+                    "yarn.scheduler.capacity.root.applications.prod.capacity" : "99",
+                    "yarn.scheduler.capacity.root.applications.prod.maximum-applications" : "1000000",
+                    "yarn.scheduler.capacity.root.applications.prod.maximum-capacity" : "100",
+                    "yarn.scheduler.capacity.root.applications.prod.minimum-user-limit-percent" : "100",
+                    "yarn.scheduler.capacity.root.applications.prod.ordering-policy" : "fair",
+                    "yarn.scheduler.capacity.root.applications.prod.ordering-policy.fair.enable-size-based-weight" : "false",
+                    "yarn.scheduler.capacity.root.applications.prod.priority" : "0",
+                    "yarn.scheduler.capacity.root.applications.prod.state" : "RUNNING",
+                    "yarn.scheduler.capacity.root.applications.prod.user-limit-factor" : "100",
+                    "yarn.scheduler.capacity.root.applications.queues" : "dev,prod",
+                    "yarn.scheduler.capacity.root.applications.state" : "RUNNING",
+                    "yarn.scheduler.capacity.root.capacity" : "100",
+                    "yarn.scheduler.capacity.root.default.acl_administer_queue" : "*",
```

Comment:
This line grants everyone to the rights to administer them default.
(Same issue with the other flavors.)